assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -65,15 +65,15 @@ function useInsertElement() {
   const insertElement = useCallback(
     (type, props) => {
       const element = createElementForCanvas(type, props);
-      const { id: elementId, resource } = element;
+      const { id, resource } = element;
       addElement({ element });
       if (resource) {
-        backfillResource(resource, elementId);
+        backfillResource(resource);
       }
       // Auto-play on insert.
       if (type === 'video') {
         setTimeout(() => {
-          const videoEl = document.getElementById(`video-${elementId}`);
+          const videoEl = document.getElementById(`video-${id}`);
```

Comment:
If we are using the video's id now, what if there are two element with same id on one page?

Reply:
This is just a rename, it's the element UUID.

Reply:
Oh I see. Nevermind.